### DISCUSSION TITLE
Error importing from github

### DISCUSSION THREAD

**Discussion by pokke1** [smartcontractkit/full-blockchain-solidity-course-py#610] [Q&A] [answered]
This is the FundMe contract, I get a file not found error,even tough if I copy paste the path of the file in the explorer I get the right contract,I'm confused.
```Solidity
`// SPDX-License-Identifier: MIT

pragma solidity ^0.6.6;

import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";

contract FundMe {
    using SafeMathChainlink for uint256;

    mapping(address => uint256) public addressToAmountFunded;
    address[] public funders;
    address public owner;
    AggregatorV3Interface public priceFeed;

    // if you're following along with the freecodecamp video
    // Please see https://github.com/PatrickAlphaC/fund_me
    // to get the starting solidity contract code, it'll be slightly different than this!
    constructor(address _priceFeed) public {
        priceFeed = AggregatorV3Interface(_priceFeed);
        owner = msg.sender;
    }

    function fund() public payable {
        uint256 mimimumUSD = 50 * 10**18;
        require(
            getConversionRate(msg.value) >= mimimumUSD,
            "You need to spend more ETH!"
        );
        addressToAmountFunded[msg.sender] += msg.value;
        funders.push(msg.sender);
    }

    function getVersion() public view returns (uint256) {
        return priceFeed.version();
    }

    function getPrice() public view returns (uint256) {
        (, int256 answer, , , ) = priceFeed.latestRoundData();
        return uint256(answer * 10000000000);
    }

    // 1000000000
    function getConversionRate(uint256 ethAmount)
        public
        view
        returns (uint256)
    {
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUsd = (ethPrice * ethAmount) / 1000000000000000000;
        return ethAmountInUsd;
    }

    function getEntranceFee() public view returns (uint256) {
        // mimimumUSD
        uint256 mimimumUSD = 50 * 10**18;
        uint256 price = getPrice();
        uint256 precision = 1 * 10**18;
        return (mimimumUSD * precision) / price;
    }

    modifier onlyOwner() {
        require(msg.sender == owner);
        _;
    }

    function withdraw() public payable onlyOwner {
        msg.sender.transfer(address(this).balance);

        for (
            uint256 funderIndex = 0;
            funderIndex < funders.length;
            funderIndex++
        ) {
            address funder = funders[funderIndex];
            addressToAmountFunded[funder] = 0;
        }
        funders = new address[](0);
    }
}
```
<img width="536" alt="Error_file_not_found" src="https://user-images.githubusercontent.com/96388096/147015079-a106bda5-a120-45d5-a751-5f148806f6fd.png">
`

**Comment by pokke1** [comment]
I'm saying the file "C:/Users/aless/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol " actually exist , I don't understand how it doesn't find it.
Also I pm downloaded "smartcontractkit/chainlink-brownie-contracts@1.1.1"

**Comment by cromewar** [comment, marked as answer]
Hello @pokke1 try deleting the entire `.brownie` folder and try to compile again, also change the version of the dependency on brownie-config to 0.3.0 which is actually higher.

**Reply by pokke1** [reply]
> Hello @pokke1 try deleting the entire `.brownie` folder and try to compile again, also change the version of the dependency on brownie-config to 0.3.0 which is actually higher.

thank you, it started downloading something but then I get this error :
`WARNING: Unable to compile smartcontractkit/chainlink-brownie-contracts@0.3.0 due to a PermissionError - you may still be able to import sources from the package, but will be unable to load the package directly.`

How can I solve it ?

**Reply by cromewar** [reply]
Well this is actually tricky to answer is it is not an error, this is just saying that all the repository will not be compiled as there are some files which needs admin privileges or are not originally from that repo, but you will be able to use it anyways as reference.

Explanation, if there is a reference to let's se Openzeppelin or Chainlink onside that repo, you will not compile it because you don't have access directly, but you still will be able to use it.
